src/ToDoForm.jsx

```diff
@@ -1,21 +1,35 @@
-import { useRef } from 'react';
+import { useState, useRef } from 'react';
 
 function ToDoForm ({onAddTodo}) {
+    const [workingTodoTitle, setWorkingTodoTitle] = useState("");
     const todoTitleInput = useRef("");
     
-    function handleAddTodo(event){
-        event.preventDefault();
-        const title = event.target.title.value;
-        event.target.title.value = "";
-        onAddTodo(title);
+    // local handleAddTodo function performs the following:
+    function handleAddTodo(title){
+        // prevent page re-render
+        title.preventDefault();
```

Comment:
Using the word "title", makes it seem like `handleAddTodo` should take in text. To keep the function easy to understand, make sure that the argument name is descriptive of what the function expects. In this case, we can still use `event`.